hybrid/ddc_compartment_switching/ddc_compartment_switching.c

```diff
@@ -0,0 +1,42 @@
+/***
+ * This example shows an intermediate level of compartment creation.
+ * It allocates a block of memory (5000 bytes) but only the first
+ * 2000 are accessible by `compartment_simple_fun`.
+ *
+ * This program does not raise any exception because we verify we can
```

Comment:
This sentence isn't quite right (it's not "because we verify the load"). "This program does not raise an exception because it explicitly performs loads/stores within the compartment." might be better?

Reply:
Done in c7507be.